### DISCUSSION TITLE
Neestream

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#143] [Show and tell]
https://live.neestream.com/neestream_01/anandtv/playlist.m3u8 (doesn't work)
https://live.neestream.com/neestream_01/nee_cinema/playlist.m3u8
https://live.neestream.com/neestream_01/asianetnews/playlist.m3u8
https://live.neestream.com/neestream_01/we/playlist.m3u8
https://live.neestream.com/neestream_01/janam/playlist.m3u8
https://live.neestream.com/neestream_01/kairalitv/playlist.m3u8
https://live.neestream.com/neestream_01/kairali_news/playlist.m3u8
https://live.neestream.com/neestream_01/jaihind/playlist.m3u8
https://live.neestream.com/neestream_01/mediaone/playlist.m3u8
https://live.neestream.com/neestream_01/mathrubhumi/playlist.m3u8

.net alt

https://live.neestream.net/neestream_01/anandtv/index.m3u8?token=693400
https://live.neestream.net/neestream_01/neecinema/index.m3u8?token=693400
https://live.neestream.net/neestream_01/asianetnews/index.m3u8?token=693400
https://live.neestream.net/neestream_01/kairalitv/index.m3u8?token=693400
https://live.neestream.net/neestream_01/kairaliwe/index.m3u8?token=693400
https://live.neestream.net/neestream_01/kairalinews/index.m3u8?token=693400
https://live.neestream.net/neestream_01/janamtv/index.m3u8?token=693400
https://live.neestream.net/neestream_01/jaihindtv/index.m3u8?token=693400
https://live.neestream.net/neestream_01/mediaone/index.m3u8?token=693400
https://live.neestream.net/neestream_01/mathrubhuminews/index.m3u8?token=693400

**Comment by what-a** [comment]
Playing fine here in California. Tested 'em all.

**Reply by BellezaEmporium** [reply]
I'll have to find a listing from their channels, so that I can get the links for all of them.
